backend/codesirius/authentication/models/verification.py

```
@@ -81,8 +93,22 @@ def save(self, *args, **kwargs):
+                topic="email",
+                value=json.dumps(
+                    {
+                        "email": self.user.email,
+                        "verification_code": self.code,
+                        "username": self.user.username,
+                        "user_id": str(self.user.id),
+                    }
+                ),
+                callback=delivery_report,
+            )
+        except Exception as e:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)